### PR TITLE
Adding mv-expand limit.

### PR DESCRIPTION
**Pull request by MonikaReddy-MSFT**
2403110050002452 

https://dev.azure.com/AzureMonitorPOD/Azure%20Monitoring%20Documentation/_boards/board/t/Azure%20Monitoring%20Documentation%20Team/Issues/?workitem=216

# Thank you for contributing to Azure Data Explorer documentation

## Fill out these items before submitting your pull request:

If you are working internally at Microsoft:

- **Provide a link to an Azure DevOps Boards work item that tracks this feature/update.**

- **Who is your Docs team contact?** \@mention them individually tag them and let them review the PR before signing off.

## For internal Microsoft contributors, check off these quality control items as you go

- [ ] 1. **Check the Acrolinx report:** Make sure your Acrolinx Total score is **above 80 minimum** (higher is better) and with no spelling issues. Acrolinx ensures we are providing consistent terminology and using an appropriate voice and tone, and helps with localization.

- [ ] 2. **Successful build with no warnings or suggestions**: Review the build status to make sure **all files are green** (Succeeded).

- [ ] 3. **Preview the pages:**: Click each **Preview URL** link to view the rendered HTML pages on the review.learn.microsoft.com site to check the formatting and alignment of the page. Scan the page for overall formatting, and look at the parts you edited in detail.

- [ ] 4. **Check the Table of Contents:** If you are adding a new markdown file, make sure it is linked from the table of contents.

- [ ] 5. **#sign-off to request PR review and merge**: Once the pull request is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the Pull Request. If you need to cancel that sign-off, type `#hold-off` instead. *Signing off means the document can be published at any time.* Note, this is a formatting and standards review, not a technical review.

## Merge and publish

- After you `#sign-off`, there is a separate PR Review team that will review the PR and describe any necessary feedback before merging. 
- The review team will use the comments section in the PR to provide feedback if changes are needed. Address any blocking issues and sign off again to request another review.
- Once all feedback is resolved, you can `#sign-off` again. The PR Review team reviews and merges the pull request into the specified branch (usually the *main( branch or a *release-branch*).
- From the *main* branch, the change is merged into the *live* branch several times a day to publish it to the public learn.microsoft.com site.

### CODE REVIEW

File: data-explorer/kusto/query/mv-expand-operator.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Expands multi-value dynamic arrays or property bags into multiple records.
 that pack multiple values into a single [dynamic](./scalar-data-types/dynamic.md)-typed
 array or property bag, such as `summarize` ... `make-list()` and `make-series`.
 Each element in the (scalar) array or property bag generates a new record in the
-output of the operator. All columns of the input that aren't expanded are duplicated to all the records in the output.
+output of the operator. All columns of the input that aren't expanded are duplicated to all the records in the output. mv-expand is limited to 2,000 records.
```
Comment (Jak-MS) [inline data-explorer/kusto/query/mv-expand-operator.md:16]:
```suggestion
output of the operator. All columns of the input that aren't expanded are duplicated to all the records in the output. `mv-expand` is limited to 2,000 records.
```

**Comment by Jak-MS** [comment]
@shsagir 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by YaelSchuster** [comment]
Have asked @sloutsky to review

**Comment by YaelSchuster** [comment]
Hi @MonikaReddy-MSFT  - according to the product team, this change isn't correct. Can you explain what's the reason behind it? Also, MSFT employees are supposed to make PRs exclusively on the private repo.

#please-close

**Pull request closed without merging**